node-operators/oracle-nodes/node-chainlink.md

```diff
@@ -18,206 +18,262 @@ description: 如何在Moonbeam网络设置Chainlink预言机节点为智能合
+
+    ![Chainlink address](/images/node-operators/oracle-nodes/chainlink/chainlink-node-3.png)
 
 下一步，部署预言机合约，它是区块链和节点之间的中间件。合约将发送包含所有必要信息的事件信息，并被预言机节点读取。然后节点将完成请求，并将所请求的数据写入调用者的合约。
 
-预言机合约的源代码可以在Chainlink的官方[GitHub repository](https://github.com/smartcontractkit/chainlink/tree/develop/contracts/src/v0.6)中找到。在本示例中，我们将使用Remix来与Moonbase Alpha交互并部署合约。在Remix环境下，可以复制以下代码：
+预言机合约的源代码可以在Chainlink的官方[GitHub repository](https://github.com/smartcontractkit/chainlink/tree/develop/contracts/src/v0.6/Oracle.sol){target=blank}中找到。在本示例中，我们将使用Remix来与Moonbase Alpha交互并部署合约。在[Remix](https://remix.ethereum.org/){target=blank}环境下，可以复制以下代码：
 
 ```
 pragma solidity ^0.6.6;
 
-import "https://github.com/smartcontractkit/chainlink/evm-contracts/src/v0.6/Oracle.sol";
+import "@chainlink/contracts/src/v0.6/Oracle.sol";
 ```
 
-编译好合约后，进入“Deploy and Run Transactions”标签，输入Link代币地址，并部署合约。部署完成后，复制合约地址。
+编译合约后，您可以通过以下步骤部署并与合约进行交互：
+
+1. 进入**Deploy and Run Transactions**标签
+
+2. 请确保已经选择**Injected Web3**，并将MetaMask连接至Moonbase Alpha
+
+3. 验证您的地址被选择
+
+4. 输入LINK Token地址，并点击**Deploy**以部署合约。MetaMask将弹出弹窗，您可以确认交易
+
+5. 部署完成后，在**Deployed Contracts**板块复制合约地址。
 
 ![Deploy Oracle using Remix](/images/node-operators/oracle-nodes/chainlink/chainlink-node-4.png)
 
-最后，绑定预言机节点和预言机智能合约。节点可以捕获发送到特定预言机合约的请求，但只有被授权（即绑定）的节点才能完成这一任务。
+最后，绑定预言机节点和预言机智能合约。节点可以捕获发送到特定预言机合约的请求，但只有被授权（即绑定）的节点才能完成这一任务。请执行以下步骤以绑定预言机节点和智能合约：
 
-使用预言机合约中的`setFulfillmentPermission()`函数进行授权，这一操作需要两个参数：
+1. 使用预言机合约中的`setFulfillmentPermission()`函数进行授权，输入您想要与合约绑定的节点地址
 
- - 需要绑定到合约的节点地址（即上一步的内容）
- - 设置绑定状态下的布尔值，在本示例中设置为`true`
+2. 在`_allowed`字段中您可以设置绑定状态下的布尔值逻辑，本例输入`true`
 
-我们可以使用在Remix上部署的合约实例来完成这一操作，并在视图函数`getAuthorizationStatus()`中输入预言机节点地址，检查预言机节点是否获得授权。
+3. 点击**transact**发送请求。MetaMask将弹出弹窗，您可以确认交易
+
+4. 通过视图函数`getAuthorizationStatus()`检查预言机节点是否获得授权，传入预言机节点地址
 
 ![Authorize Chainlink Oracle Node](/images/node-operators/oracle-nodes/chainlink/chainlink-node-5.png)
 
-## 在预言机节点上创建任务 {: #create-job-on-the-oracle-node } 
+## 创建Job {: #creating-a-job } 
 
-Chainlink预言机配置的最后一步就是创建任务。请参阅[Chainlink官方文档](https://docs.chain.link/docs/job-specifications)：
+Chainlink预言机配置的最后一步就是创建Job。请参阅[Chainlink官方文档](https://docs.chain.link/docs/job-specifications){target=_blank}：
 
 > 任务参数包含了一系列任务，节点必须执行这些任务才能获得最终结果。一条参数包含至少一个启动程序和一个任务（此前已详细讨论）。参数使用标准化JSON进行定义，实现人类可读，并能够轻易被Chainlink节点所分析。
 
-如果将预言机看作API服务，那么任务就是其中一个函数，我们调用这个函数并获得返回结果。创建第一个任务，请来到节点的[“Jobs”板块](http://localhost:6688/jobs)，并点击“New Job”。
+如果将预言机看作API服务，那么Job就是其中一个函数，我们调用这个函数并获得返回结果。请执行以下步骤创建您的第一个Job：
 
-![Chainlink Oracle New Job](/images/node-operators/oracle-nodes/chainlink/chainlink-node-6.png)
+1. 进入[您节点的Job板块](http://localhost:6688/jobs){target=_blank}
 
-下一步，粘贴以下JSON代码。创建的任务将请求ETH目前的的美金价格。请务必输入您的预言机合约地址（`YOUR_ORACLE_CONTRACT_ADDRESS`）。
+2. 点击**New Job**
 
-```
+![Chainlink oracle New Job](/images/node-operators/oracle-nodes/chainlink/chainlink-node-6.png)
+
+下一步，您可以创建新的Job：
+
+1. 粘贴以下JSON代码。创建的任务将请求ETH目前的的美金价格。
```

Comment:
looks like this could be missing USD but not really sure

English version: 
` 1. Paste the following JSON. This will create a job that will request the current ETH price in USD`

Reply:
used google translate hehe looks all good 👍